Return Point

terrancewoodard opened the pull request:
This lets the user adds a return point for the Sky Wrangler UAV drone

terrancewoodard committed:
Return Point

This lets the user adds a return point for the Sky Wrangler UAV drone

dlech reviewed (changes requested):
There is a major thing missing here. In addition to getting input from the user, we also need to send the information to the skywrangler-web-server when we launch the mission.

terrancewoodard committed:
Return Points Updates

I have updated the names of the "return points" to "ReturnPointLatitude" and "ReturnPointLongitude". I have also connected the information to the web server.

dlech reviewed (changes requested):
I still don't see the part where we send the new point to the skywrangler-web-server.

terrancewoodard committed:
Return Point Updates #2

I have renamed the return points and also added another value so that the information can connect to the web server.

terrancewoodard committed:
Update Names For Hooks

we changed Return into return (changed the capitalization).

terrancewoodard committed:
changing the casing of the return point values

I went through the program and changed the casing around to match each other.

terrancewoodard committed:
changing the casing

I went through the files and changed the casing to the correct format.

terrancewoodard committed:
Updated the handle sign

I updated the handle sign to change to its corresponding value instead of having it assigned to the same value.
